src/arch/riscv/isa/formats/vector_arith.isa

```diff
@@ -79,38 +79,45 @@ let {{
             uint32_t ei = i + vtype_VLMAX(vtype, vlen, true) * this->microIdx;
             ''' + code
 
-    def wideningOpRegisterConstraintChecks(code):
-        return '''
-            const uint32_t num_microops = 1 << std::max<int64_t>(0, vtype_vlmul(machInst.vtype8) + 1);
+    def wideningOpRegisterConstraintChecks(code, src2_dw):
+        constraint_checks = '''
+            const uint32_t num_microops = 1 << std::max<int64_t>(0, vlmul + 1);
```

Comment:
Is this specific to the case where `SEW` is widened to `2*SEW`?

Reply:
Yes, indeed. As of now, no instruction widens to more than 2*SEW, but let me try to generalize it.